index.html

```diff
@@ -46,9 +47,259 @@
+                                <div class="see-project"><button class="see-project-btn">See Project</button></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-item">
+                        <div class="grid-container work-item-grid">
+                            <div class="work-image"><img src="./images/proj-5.jpg"></div>
+                            <div class="grid-container work-specs">
+                                <div class="work-spec-items"><h3>English Dictionary</h3></div>
+                                <div class="recent-work-det">
+                                    <ul>
+                                        <li>Ruby</li>
+                                        <li>HTML</li>
+                                        <li>CSS</li>
+                                        <li>PHP</li>
+                                    </ul>
+                                </div>
+                                <div class="see-project"><button class="see-project-btn">See Project</button></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-item">
+                        <div class="grid-container work-item-grid">
+                            <div class="work-image"><img src="./images/proj-6.jpg"></div>
+                            <div class="grid-container work-specs">
+                                <div class="work-spec-items"><h3>System Administration</h3></div>
+                                <div class="recent-work-det">
+                                    <ul>
+                                        <li>Ruby</li>
+                                        <li>HTML</li>
+                                        <li>CSS</li>
+                                        <li>PHP</li>
+                                    </ul>
+                                </div>
+                                <div class="see-project"><button class="see-project-btn">See Project</button></div>
+                            </div>
+                        </div>
+                    </div>
+                    
+                </div>
+             </div>
+    </section>
+
+    <!--- ABOUT ME -->
+    <section class="about-me-container">
+        <div class="grid-container about-me-sub">
+
+            <div class="grid-item about-me-grid">
+                <div class="about-me-heading"><h1>About Me</h1></div>
+            </div>
+
+            <div class="grid-item me-detail-grid">
+                <div class="detail-para"><p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
+            </div>
+
+            <div class="grid-item see-btn-grid">
+                <button class="see-project-btn">Get my Resume</button>
+            </div>
```

Comment:
- [ ]   Good job @ehmaddd 😃, but you need to fix some parts such as (`the margin`, `font size`, `and color`) and you need to fix your background image to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=DlZ5PelwOdZvAKL2-0) 👍
> The h1 color: #172B4D; font-weight: 700; font-size: 40px;
> The p color: #42526E; font-weight: 400; font-size: 20px;

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![aboutfg](https://user-images.githubusercontent.com/57408419/219854731-57c56279-4080-4f5b-b69c-6f5f434823b6.jpg)| |![about](https://user-images.githubusercontent.com/57408419/219854754-077da948-2e13-4eb5-bd8d-7f9ae6b9e604.jpg)| |

Reply:
Also fixed this according to the requirement